react/components/NotificationBar/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { memo } from 'react'
```

Comment:
Since this is a totally new component why don't you do it in typescript?

Reply:
I don't think it's a good approach mix both in the same app.